src/test/scala/is/hail/variant/vsm/VSMSuite.scala

```diff
@@ -58,23 +58,19 @@ class VSMSuite extends SparkSuite {
       Array("S1", "S2"),
       Annotation.emptyIndexedSeq(2),
       Annotation.empty,
-      TStruct(
+      sSignature = TStruct(
         "inner" -> TStruct(
           "thing1" -> TString),
-        "thing2" -> TString),
-      TStruct.empty,
-      TStruct.empty)
+        "thing2" -> TString))
```

Comment:
The `vaSignature` defaults to `TStruct.empty` but here the variant annotations have the form `Annotation(Annotation("yes"), "yes")`, so for consistency I think the type should be something like `TStruct("q1" -> TStruct("q2" -> TString))`.

Also, you may as well remove the variables `r1`, `r2` and `r3` below and just write `val va1 = Annotation(Annotation("yes"), "yes")`, etc.